shell.py

```diff
@@ -12,10 +14,30 @@ def write(lba, data):
     """write"""
     pass
 
-@cli.command(name="read")
-@click.argument('lba')
-def read(lba):
-    pass
+def call_system(cmd:str):
+    try:
+        result = subprocess.run("dir", shell=True, capture_output=True, text=True, encoding='cp949', check=True)  # or 'euc-kr'
+    except Exception:
+        #TODO : Exception에 대한 처리 없이 오류 발생한 returncode를 리턴하는 것으로 대체.
+        ...
+    return result.returncode
+
+
+def read_result_file(filename):
+    line = None
+    with open(filename, 'r' ) as f: #TODO encoding 확인 필요
+        line = f.read()
+    return line
+
+def read(lba, filename = 'ssd_output.txt'):
+    #TODO lba 범위 확인 & 에러 처리
+
+    status = call_system(f'python ssd.py R {lba}')
+    if status >= 0:
+        read_data = read_result_file(filename)
+        #TODO : lba 자릿수 고정.
+        result_str = f'[READ] LBA {lba} : {read_data}'
+        print(result_str)
```

Comment:
```suggestion
        print(f'[READ] LBA {lba} : {read_data}')
```
이렇게 가는게 조금 더 깔끔할것같습니다!